frontend/src/data_context_global.tsx

```diff
@@ -139,6 +139,7 @@ export interface Breeding {
 export interface ExtendedBreeding extends Breeding {
   mother_name: string | null | undefined;
   father_name: string | null | undefined;
+  individuals: Array | null | undefined;
```

Comment:
@HenrikeW not sure if this is Needed?

Reply:
Yes I think it's needed, but it caused type errors in other places. Fixed now.